tls-core/src/prf/mod.rs

```diff
@@ -6,9 +7,32 @@ mod utils;
 pub use master::PrfMaster;
 pub use slave::PrfSlave;
 
+use master::{
+    KeInit, MasterKe1, MasterKe2, MasterKe3, MasterMs1, MasterMs2, MasterMs3, MasterSetup,
+};
+use slave::{SlaveKe1, SlaveKe2, SlaveMs1, SlaveMs2, SlaveMs3};
+
+#[derive(Copy, Clone)]
+pub enum PRFMessage {
+    MasterSetup(MasterSetup),
```

Comment:
This isn't a message, we should pull it out and pass it in either the constructor or a standalone `fn setup(...)` method